### PR TITLE
Ping server fatal error

### LINKED ISSUE #80: Fatal error when PANTHEON_INDEX_PORT or PANTHEON_INDEX_HOST environment variables are not configured
From [PR #74](https://github.com/pantheon-systems/solr-power/pull/74)
> PHP Fatal error:  Call to a member function ping() on a non-object
> `get_solr()` return `NULL` until Solr server configuration has been done.

We should check if the environment variables are set and, if not, display an WordPress dashboard warning to the user rather than having a fatal error.

### PR DESCRIPTION
**Pull request by ataylorme**
Fixes #80

### CODE REVIEW

**Comment by pantheon-circleci** [comment]
@ataylorme, thanks for your PR! By analyzing the blame information on this pull request, we identified @allan23, @calevans and @erik-pantheon to be potential reviewers

**Comment by ataylorme** [comment]
@tekezo This is the updated pull request to add a warning message and not allow the plugin to be activated if the Solr Pantheon environment variables are not set as discussed in #74.

I've tested this but would like you to test/confirm as well before merging.

Thanks!

**Comment by tekezo** [comment]
@ataylorme Thank you! This change works like a charm!

**Comment by ataylorme** [comment]
> @ataylorme Thank you! This change works like a charm!

@tekezo - great! Thank you for reporting the bug. I'll get this merged in and put a release out to WordPress.org by the end of the week.

File: solr-power.php [hunk 1]
```diff
@@ -76,6 +76,41 @@ function solr_power_PHP_admin_notice() {
...
+function solr_power_env_variables_admin_notice() {
+	?>
+	<div class="error">
+		<p>
+			<?php
+			if ( isset( $_GET['activate'] ) ) {
+				unset( $_GET['activate'] );
+
+				_e(
+					'The Solr Power plugin requires environment varaibles for <pre>PANTHEON_INDEX_HOST</pre> and <pre>PANTHEON_INDEX_PORT</pre> to function properly.<br />' .
```
Comment (allan23) [inline solr-power.php]: Small typo: 'varaibles' should be 'variables'

**Pull request merged**